src/Uno.UI/Microsoft/UI/Xaml/Controls/ProgressRing/ProgressRing.cs

```diff
@@ -80,6 +82,20 @@ private void SetAnimatedVisualPlayerSource()
 				_player.Source = animatedVisualSource;
 				ChangeVisualState();
 			}
+			else if (_player != null && _layoutRoot != null)
+			{
+				// If we have a _player, it means we're having a ControlTemplate relying
+				// on it.  In this case, the Uno.UI.Lottie reference is required for the
+				// rendering of the ProgressRing.
+
+				var txt = new TextBlock
+				{
+					Text = "⚠️ Uno.UI.Lottie missing ⚠️",
```

Comment:
```suggestion
					Text = "⚠️ To use the ProgressRing, add the Uno.UI.Lottie nuget package ⚠️",
					TextWrapping="Wrap"
```

Reply:
The goal here is to have a very short message because it's usually 32px large.